tests/test_cli.py

```diff
@@ -83,6 +84,71 @@ def test_cve_list():
         )
 
 
+def test_cve_create():
+    cve_id = "CVE-2001-0635"
+    cna_json = {
```

Comment:
Is this supposed to represent a minimal CNA container? Can we call it `cna_container_data` (it's not JSON per se)?